package/setup.py

```diff
@@ -494,11 +494,12 @@ def dynamic_author_list():
           classifiers=CLASSIFIERS,
           cmdclass=cmdclass,
           requires=['numpy (>=1.10.4)', 'biopython', 'mmtf (>=1.0.0)',
-                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)', 'joblib'],
+                    'networkx (>=1.0)', 'GridDataFormats (>=0.3.2)', 'joblib',
+                    'scipy', 'matplotlib (>=1.5.1)'],
           # all standard requirements are available through PyPi and
           # typically can be installed without difficulties through setuptools
           setup_requires=[
-              'numpy>=1.9.3',
+              'numpy>=1.10.4',
```

Comment:
did you add this bump to the CHANGELOG? I know we did it a while again and forgot to change setup.py